README.md

```diff
@@ -206,3 +206,9 @@ Semantic UI is an open source UI framework.
 
 - [Tailwind CSS](https://tailwindcss.com/)
 Tailwind CSS is a utility-first CSS framework for rapidly building custom user interfaces.
+
+## Software Development FlowChart 
```

Comment:
Let's remove this category and place the Miro submission into the Misc. category instead.